checkov/terraform/checks/resource/kubernetes/PodSecurityContext.py

```diff
@@ -0,0 +1,52 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class PodSecurityContext(BaseResourceCheck):
+
+    def __init__(self):
+        # CIS-1.5 5.7.3
+        name = "Apply security context to your pods and containers"
+        # Security context can be set at pod or container level.
+        id = "CKV_K8S_29"
+
+        supported_resources = ['kubernetes_pod', 'kubernetes_deployment', 'kubernetes_daemonset']
+        categories = [CheckCategories.GENERAL_SECURITY]
```

Comment:
why not `CheckCategories.KUBERNETES`?

Reply:
because every check for the provider would be that.

Reply:
Ive added type checking